macros/edr/data_monitoring/monitors_query/table_monitoring_query.sql

```diff
@@ -1,78 +1,33 @@
-{% macro table_monitoring_query(monitored_table_relation, timestamp_column, min_bucket_start, table_monitors, freshness_column, where_expression, time_bucket) %}
+{% macro table_monitoring_query(monitored_table_relation, timestamp_column, min_bucket_start, table_monitors, where_expression, time_bucket, metric_args) %}
 
     {% set full_table_name_str = elementary.quote(elementary.relation_to_full_name(monitored_table_relation)) %}
 
+    with monitored_table as (
+        select * from {{ monitored_table_relation }}
+        {% if where_expression %}
+        where {{ where_expression }}
+        {% endif %}
+    ),
+
     {% if timestamp_column %}
-        with buckets as (
+        buckets as (
             select edr_bucket_start, edr_bucket_end from ({{ elementary.complete_buckets_cte(time_bucket) }})
             where edr_bucket_start >= {{ elementary.cast_as_timestamp(min_bucket_start) }}
         ),
 
-        filtered_monitored_table as (
+        time_filtered_monitored_table as (
             select *,
                    {{ elementary.get_start_bucket_in_data(timestamp_column, min_bucket_start, time_bucket) }} as start_bucket_in_data
```

Comment:
in BQ the complied CTE that I get is:
```
ime_filtered_monitored_table as (
            select *,
                   cast(

        datetime_add(
            cast( '2023-01-02 00:00:00+00:00' as datetime),
        interval cast(
      floor(

    datetime_diff(
        cast(cast(timestamp as TIMESTAMP) as datetime),
        cast('2023-01-02 00:00:00+00:00' as datetime),
        day
    )
```
and it raise the following error in runtime:
`Invalid datetime string "2023-01-02 00:00:00+00:00"`
removing `+00:00` manually solved the issue.
It relevant only for `datetime` casting, for timestamp no change required